.circleci/config.yml

```diff
@@ -322,14 +402,27 @@ jobs:
 workflows:
   build-and-test:
     jobs:
+      - deploy-infrastructure-dev
```

Comment:
IIRC this was put first for quicker feedback during testing, but would it make sense for this to run after the test jobs?

Additionally, we may want to apply filters to this job as well so it doesn't run on every commit and cause conflicts between PRs. If we go that route we would also want to update the Github Action for label based deploys to call the `deploy-infrastructure-dev` job before calling the deploy job. (could be a separate issue)

Reply:
This should have been removed yeah. The job should only run after the deploy jobs run, which are filtered by branches.

Reply:
```suggestion
```

Reply:
So I updated it to only run on merges to `raft-review`, and the staging deployment which runs on merge to `raft-tdp-main`. 

We don't use the `raft-review` branch currently, I was following the existing code in the deploy job. @jtwillis92 do we know what branch we intend to use for the future `dev` space when this is merged and we have a staging environment?

Reply:
The plan was after https://github.com/raft-tech/TANF-app/pull/1056 we would do all our deployments to the dev space based off assigning labels. At that point we could remove the deploy-infrastructure-dev and deploy-dev steps from the workflow or keep merging to `raft-review` as an extra option. I can't think of any technical reason we would need to keep it around though.